### PR TITLE
IMDB Clone

### PR DESCRIPTION
**Pull request by karanmann**
Project Movies
Making a multipage React.Js site (Single Page Application) by fetching data from FreeMovieDB.

Tech Used
React.JS, JSX, CSS

The problem
I faced problems with the image loading before the fetch completed causing a 404 error.

View it live
https://imdb-clone-site.netlify.app/

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,188 @@
...
+@media (min-width: 900px) {
+  .movie-page-content {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .movie-page-text-content {
+    margin: 50px;
+  }
+}
```
Comment (MindstormingAB) [inline code/src/index.css:186]: Great that you have a media query for the desktop view. It works well on tablet but I miss a mobile view. 📵

**Review by MindstormingAB — CHANGES_REQUESTED** [review]
Well done Karan! You seem to deal easily with useEffect, useState, params and BrowserRouter, fetch and API.
In addition to the requirements, you made the layout working on tablets and desktops.
Before I can approve your project, I'd like you to:
- check that your code is written correctly: mind the indentation and make sure that you stick to one approach when it comes to semi-colons (everywhere or mandatory only) and quotation marks (' or ").
- adapt the size of your backdrop image
- remove the unused code/styling/files
- make the movie page responsive on mobile
- make sure that the "Know more" link will open in a different window

File: code/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import TechnigoLogo from '../img/technigo.svg'
```
Comment (EmOlsz) [inline code/src/components/Footer.js:3]: Uppercase names are reserved for components only. Here we should rather have :

```
import technigoLogo from '../img/technigo.svg'
```

File: code/src/components/MoviePage.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }, [movieId])
+
+  useEffect(() => {
+    if(movie.id) setLoading(false)
+  }, [movie]);
+
+  
+
+  return (
+    <>
```
Comment (EmOlsz) [inline code/src/components/MoviePage.js:26]: No need for React fragment here since you return only one element - `.movie-page-background`

**Pull request closed without merging**